.github/workflows/nightly-copy-production_db_into_snapshot_db_and_store_dump.yml

```
@@ -9,34 +9,34 @@ permissions:
   id-token: write
 
 jobs:
-  generate-day_of_week:
+  generate-date:
     runs-on: ubuntu-latest
     outputs:
-      today_day: ${{ steps.get-day.outputs.today_day }}
+      today: ${{ steps.get-date.outputs.today }}
     steps:
-      - name: Get today's day of the week
-        id: get-day
-        run: echo "today_day=$(date +'%A')" >> $GITHUB_OUTPUT
+      - name: Get today's date
+        id: get-date
+        run: echo "today=$(date +'%F')" >> $GITHUB_OUTPUT
```

Maybe we could omit the special characters to make this more robust for storage?

```suggestion
        run: echo "today=$(date +'%Y%m%d')" >> $GITHUB_OUTPUT
```

---

We're using `+'F'` elsewhere so i left it for consistency, but if we were starting fresh I'd agree.